src/switch_core_media.c

```diff
@@ -4852,21 +4852,23 @@ static switch_status_t check_ice(switch_media_handle_t *smh, switch_media_type_t
 			switch_port_t remote_rtcp_port = engine->remote_rtcp_port;
 
 			if (remote_rtcp_port) {
```

Comment:
You can move your condition on this if condition. I would also suggest that the behavior would be to check a variable that disable rtcp rather than check if we will enable rtcp. We already taken time to support QOS in the mission portal and removing this by default may impact other customers.

Reply:
OK, we can use `telnyx_disable_rtcp`, but it should prevent a call to `switch_rtp_activate_rtcp` rather than just skipping what's conditional on `remote_rtcp_port`, as `switch_rtp_activate_rtcp` initialises all the RTCP context.